examples/source/interactivity-dynamic-content/Tictactoe.html

```diff
@@ -0,0 +1,319 @@
+<!---
+
+preview: default
+teaserImage: '/static/samples/img/teaser/tictactoe.png'
+author: garanj
+
+--->
+
+<!-- ## Introduction -->
+<!--
+This is a sample showing how to implement Tic-Tac-Toe. This includes:
+
+*   How to use `amp-state` to maintain the state of play.
+*   How to use _expressions_ to detect winning moves.
+*   How to use _actions_ and _events_ to create interative gameplay.
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>Tic-tac-toe</title>
```

Comment:
Please change to `Tic-Tac-Toe`. And could you also change the filename to Tic_Tac_Toe.html to match the other filenames?

Reply:
Done!